website/versioned_docs/version-28.0/CLI.md

```diff
@@ -204,7 +204,18 @@ Alias: `-e`. Use this flag to show full diffs and errors instead of a patch.
 
 ### `--filter=<file>`
 
-Path to a module exporting a filtering function. This method receives a list of tests which can be manipulated to exclude tests from running. Especially useful when used in conjunction with a testing infrastructure to filter known broken.
+Path to a module exporting a filtering function. This asynchronous function receives a list of test paths which can be manipulated to exclude tests from running by returning an object with the "filtered" property. Especially useful when used in conjunction with a testing infrastructure to filter known broken, e.g.
+
+```javascript
+// my-filter.js
```

Comment:
Not sure if file name is useful here, but better like this:

```suggestion
```js title="my-filter.js"
```

Reply:
I did not know it existed, thanks!